### PR TITLE
Added tests for the Custom CRS capabilities 

### PR DESCRIPTION
**Pull request by emacthecav**

### CODE REVIEW

File: test/src/test/java/mil/nga/giat/geowave/test/landsat/CustomCRSLandsatIT.java [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+					new StringReader(
+							ingestOptions.getCoverageName()),
+					new Configuration());
+		}
+
+	}
+
+	@GeoWaveTestStore(value = {
+		GeoWaveStoreType.ACCUMULO,
+	// GeoWaveStoreType.BIGTABLE,
```
Comment (emacthecav) [inline test/src/test/java/mil/nga/giat/geowave/test/landsat/CustomCRSLandsatIT.java]: I'll fix this

File: test/src/test/java/mil/nga/giat/geowave/test/landsat/CustomCRSLandsatIT.java [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+	private static final int MIN_PATH = 198;
+	private static final int MAX_PATH = 199;
+	private static final int MIN_ROW = 36;
+	private static final int MAX_ROW = 37;
+	private static final double WEST = -2.2;
+	private static final double EAST = -1.4;
+	private static final double NORTH = 34.25;
+	private static final double SOUTH = 33.5;
+	// private static final double WEST = -245440;
+	// private static final double EAST = -154680;
```
Comment (emacthecav) [inline test/src/test/java/mil/nga/giat/geowave/test/landsat/CustomCRSLandsatIT.java]: Can probably take these out too

File: test/src/test/java/mil/nga/giat/geowave/test/landsat/LandsatIT.java [hunk 3]
```diff
@@ -110,9 +111,9 @@ protected void processParameters(
 
 	@GeoWaveTestStore(value = {
 		GeoWaveStoreType.ACCUMULO,
-		GeoWaveStoreType.BIGTABLE,
-		GeoWaveStoreType.CASSANDRA,
-		GeoWaveStoreType.HBASE
+	// GeoWaveStoreType.BIGTABLE,
+	// GeoWaveStoreType.CASSANDRA,
```
Comment (emacthecav) [inline test/src/test/java/mil/nga/giat/geowave/test/landsat/LandsatIT.java]: Will revert

File: test/src/test/java/mil/nga/giat/geowave/test/basic/GeoWaveBasicCustomCRSRasterIT.java [hunk 4]
```diff
@@ -0,0 +1,493 @@
...
+
+@RunWith(GeoWaveITRunner.class)
+public class GeoWaveBasicCustomCRSRasterIT extends
+		AbstractGeoWaveIT
+{
+	private static final double DOUBLE_TOLERANCE = 1E-10d;
+	@GeoWaveTestStore(value = {
+		GeoWaveStoreType.ACCUMULO,
+	// GeoWaveStoreType.BIGTABLE,
+	// GeoWaveStoreType.CASSANDRA,
```
Comment (emacthecav) [inline test/src/test/java/mil/nga/giat/geowave/test/basic/GeoWaveBasicCustomCRSRasterIT.java]: Will remove

File: core/index/src/main/java/mil/nga/giat/geowave/core/index/sfc/hilbert/PrimitiveHilbertSFCOperations.java [hunk 5]
```diff
@@ -23,6 +23,7 @@
 import com.google.uzaygezen.core.CompactHilbertCurve;
 import com.google.uzaygezen.core.FilteredIndexRange;
 import com.google.uzaygezen.core.LongContent;
+import com.google.uzaygezen.core.MathUtils;
```
Comment (rfecher) [inline core/index/src/main/java/mil/nga/giat/geowave/core/index/sfc/hilbert/PrimitiveHilbertSFCOperations.java]: delete this

File: core/index/src/main/java/mil/nga/giat/geowave/core/index/dimension/bin/BasicBinningStrategy.java [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+
+	@Override
+	public double getBinMax() {
+		return halfInterval;
+	}
+
+	public static void main(
+			String[] args ) {
+		// for (double i = -5; i <= 5; i += 1){
+		// System.out.println(Math.floor(i % 2));
```
Comment (rfecher) [inline core/index/src/main/java/mil/nga/giat/geowave/core/index/dimension/bin/BasicBinningStrategy.java]: delete this

File: extensions/datastores/cassandra/src/main/java/mil/nga/giat/geowave/datastore/cassandra/util/SessionPool.java [hunk 7]
```diff
@@ -20,18 +23,34 @@ public static synchronized SessionPool getInstance() {
...
 			final String contactPoints ) {
-		Session session = sessionCache.get(contactPoints);
-		if (session == null) {
-			session = Cluster.builder().addContactPoints(
-					contactPoints.split(",")).build().connect();
+		return getPair(
+				contactPoints).getRight();
+	}
+
+	public synchronized Pair<Cluster, Session> getPair(
```
Comment (rfecher) [inline extensions/datastores/cassandra/src/main/java/mil/nga/giat/geowave/datastore/cassandra/util/SessionPool.java]: revert this

File: test/src/test/java/mil/nga/giat/geowave/test/GeoWaveITSuite.java [hunk 8]
```diff
@@ -47,34 +50,37 @@
...
 @RunWith(GeoWaveITSuiteRunner.class)
 @SuiteClasses({
-	GeoWaveBasicSpatialVectorIT.class,
-	GeowaveCustomCRSSpatialVectorIT.class,
-	GeoWaveBasicSpatialTemporalVectorIT.class,
-	GeoWaveVectorSerializationIT.class,
-	BasicKafkaIT.class,
-	BasicMapReduceIT.class,
+//	GeoWaveBasicSpatialVectorIT.class,
+//	GeowaveCustomCRSSpatialVectorIT.class,
```
Comment (rfecher) [inline test/src/test/java/mil/nga/giat/geowave/test/GeoWaveITSuite.java]: revert this

**Pull request merged**